packages/browserify/test/generateConfig.js

```diff
@@ -42,17 +44,18 @@ test('generateConfig - ignore various refs', async (t) => {
       global.xyz
     },
     defaultConfig: false,
-  })
-  const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, {
-    resources: {
-      one: {
-        globals: {
-          xyz: true
+    expectedResult: {
+      resources: {
+        one: {
+          globals: {
+            xyz: true
+          }
         }
       }
     }
-  }, 'policy matched expected')
+  })
+  const policy = await autoConfigForScenario({ scenario })
+  scenario.checkResult(t, policy, scenario)
```

Comment:
dont use checkResult to assert an unrelated value 
just keep the original assertion